src/Wpf/Prism.Wpf/Prism.Wpf.csproj

```diff
@@ -23,7 +23,7 @@ Prism.Wpf helps you more easily design and build rich, flexible, and easy to mai
     <Reference Include="System.Configuration" />
   </ItemGroup>
 
-  <ItemGroup Condition=" $(TargetFramework.StartsWith('netcore')) OR $(TargetFramework.StartsWith('net5'))">
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('netcore')) OR $(TargetFramework.StartsWith('net6'))">
```

Comment:
@dansiegel just wondering if it would be more beneficial to use the symbols NETFRAMEWORK and NETCOREAPP to do this portion for future target framework changes to Prism WPF.

Implementation for this condition would be: $(DefineConstants.Contains('NETCOREAPP'))